### PR TITLE
Add Beginner Clue to Hiscore lookup, Hiscore plugin and Chat commands plugin

### LINKED ISSUE #8469: Clue Scroll Highscores incorrect
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**

Highscore plug in picks up wrong clue stats

**To Reproduce**
Look up player highscores, either by right clicking or typing thier name.

**Expected behavior**
clue results to match website/ true stats.

**Screenshots**

https://i.imgur.com/JIAdWtP.png
https://i.imgur.com/0Rhjlq9.png

**Environment (please complete the following information):**
 - Windows 10 
 - RuneLite version: 1.5.19.1
 - Launcher version: "unknown"
**Additional context**
Asked friends about it, they too are experiencing issues.

### PR DESCRIPTION
**Pull request by Beagon**
Fixes #8469

~Fixed the hiscore lookup, not sure if I should use this branch to fix the HiScore plugin itself to (Like add the clue icon)~
According to Adam I can put that fix/feature in this PR too.

Also fixes:
- Hiscore Plugin
- Adds `!clues beginner` to Chat command Plugin

Screenshot from 2019-04-11 16-52-01

Also: Sorry for the messy PR, I wanted to do this but didn't have the tools to do it correctly at the moment.

### CODE REVIEW

**Comment by Beagon** [comment]
There still seems to be an issue with a new entry in the HS that I can't seem to place looking at the API and the normal page. It's entry 26 (row 27) in the api call but everytime I try to find info it always returns -1.

File: http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreSkill.java [hunk 1]
```diff
@@ -52,13 +52,14 @@
 	CONSTRUCTION("Construction"),
 	BOUNTY_HUNTER_HUNTER("Bounty Hunter - Hunter"),
 	BOUNTY_HUNTER_ROGUE("Bounty Hunter - Rogue"),
+	LAST_MAN_STANDING("Last Man Standing"),
```
Comment (Adam-) [inline http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreSkill.java]: Why was this moved?

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java [hunk 2]
```diff
@@ -522,18 +522,21 @@ private String detailsHtml(HiscoreResult result, HiscoreSkill skill)
...
 					String mediumRank = (result.getClueScrollMedium().getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(result.getClueScrollMedium().getRank());
 					String hardRank = (result.getClueScrollHard().getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(result.getClueScrollHard().getRank());
 					String eliteRank = (result.getClueScrollElite().getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(result.getClueScrollElite().getRank());
 					String masterRank = (result.getClueScrollMaster().getRank() == -1) ? "Unranked" : StackFormatter.formatNumber(result.getClueScrollMaster().getRank());
 					String all = (result.getClueScrollAll().getLevel() == -1 ? "0" : StackFormatter.formatNumber(result.getClueScrollAll().getLevel()));
 					String easy = (result.getClueScrollEasy().getLevel() == -1 ? "0" : StackFormatter.formatNumber(result.getClueScrollEasy().getLevel()));
+					String beginner = (result.getClueScrollBeginner().getLevel() == -1 ? "0" : StackFormatter.formatNumber(result.getClueScrollBeginner().getLevel()));
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java]: This should be before easy

**Comment by Adam-** [comment]
This should probably have one commit to adjust for the new response from the api, one for chatcommands, and one for the hiscore plugin.

**Comment by Beagon** [comment]
I'll remember to do that next time. I'll also be

File: http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreSkill.java (hunk 1 above)
Comment (Beagon) [reply http-api/src/main/java/net/runelite/http/api/hiscore/HiscoreSkill.java]: Because LMS has changed positions in the Hiscore API and I wanted to let it reflect that.

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java (hunk 2 above)
Comment (Beagon) [reply runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java]: Agreed.
